src/main/java/org/wso2/lsp4intellij/client/languageserver/wrapper/LanguageServerWrapper.java

```diff
@@ -649,8 +649,8 @@ private void connect(String uri) {
     }
 
     /**
-     * Is the langauge server in a state where it can be resettable. Normally language server is
-     * resettable if it has timedout or has a startup error.
+     * Is the language server in a state where it can be resettable. Normally language server is
+     * resettable if it has timeout or has a startup error.
      */
     public boolean isResettable() {
```

Comment:
Restartable?

Reply:
I left it intentionally since both resettable/ restartable looked okay for me.. I'll rename it anyway :) Thanks